<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>939</width>
    <height>601</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Recordatorio de Vencimientos</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(135, 135, 135);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tablaVenci">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>891</width>
      <height>341</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <column>
     <property name="text">
      <string>Fecha Carga</string>
     </property>
     <property name="font">
      <font>
       <family>Verdana</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Nro Tramite</string>
     </property>
     <property name="font">
      <font>
       <family>Verdana</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Observacion</string>
     </property>
     <property name="font">
      <font>
       <family>Verdana</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Vencimiento</string>
     </property>
     <property name="font">
      <font>
       <family>Verdana</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Detalle</string>
     </property>
     <property name="font">
      <font>
       <family>Verdana</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="btnAnadir">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>470</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);
border-radius: 15px;</string>
    </property>
    <property name="text">
     <string>AÃ±adir</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>891</width>
      <height>501</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);

</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLineEdit" name="txtNroU">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>40</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>10</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Verdana</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Usuario</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtDias">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>180</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>16</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
    </widget>
    <widget class="QTextEdit" name="txtObs">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>110</y>
       <width>381</width>
       <height>91</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>12</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtNroT">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>40</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>16</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
    </widget>
    <widget class="QTextEdit" name="txtDetalle">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>240</y>
       <width>381</width>
       <height>101</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>12</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QDateEdit" name="dateCarga">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>110</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="displayFormat">
      <string>yyyy/M/dd</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtCerrado">
     <property name="geometry">
      <rect>
       <x>830</x>
       <y>40</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btnGuardar">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>390</y>
       <width>161</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);
border-radius: 15px;
</string>
     </property>
     <property name="text">
      <string>Guardar</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>img/salvar.png</normaloff>img/salvar.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="btnCancelar">
     <property name="geometry">
      <rect>
       <x>790</x>
       <y>460</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 0, 0);
border-radius: 15px;
</string>
     </property>
     <property name="text">
      <string>Cancelar</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);
border-radius: 10px;</string>
     </property>
     <property name="text">
      <string>Buscar</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>img/lupaa.png</normaloff>img/lupaa.png</iconset>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtUser">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>40</y>
       <width>271</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>80</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Verdana</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Fecha de Carga</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>150</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Verdana</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Recordar a X dÃ­as</string>
     </property>
    </widget>
    <widget class="QCalendarWidget" name="dateVencimiento">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>250</y>
       <width>312</width>
       <height>191</height>
      </rect>
     </property>
     <property name="accessibleName">
      <string/>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(170, 170, 255);
color: rgb(0, 0, 0);</string>
     </property>
     <property name="gridVisible">
      <bool>true</bool>
     </property>
     <property name="horizontalHeaderFormat">
      <enum>QCalendarWidget::ShortDayNames</enum>
     </property>
     <property name="verticalHeaderFormat">
      <enum>QCalendarWidget::NoVerticalHeader</enum>
     </property>
     <property name="dateEditEnabled">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>220</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Verdana</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Vencimiento</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>10</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Verdana</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Nro Tramite</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>40</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Recordatorio Abrierto</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>80</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Verdana</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Observacion</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>210</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Verdana</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Detalle</string>
     </property>
    </widget>
    <widget class="QFrame" name="frame2">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>90</y>
       <width>561</width>
       <height>231</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(85, 170, 127);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QTableWidget" name="tablaUsuarios">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>20</y>
        <width>471</width>
        <height>181</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <column>
       <property name="text">
        <string>ID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>NOMBRE USUARIO</string>
       </property>
      </column>
     </widget>
    </widget>
    <widget class="QFrame" name="frame3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>881</width>
       <height>491</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>851</width>
        <height>391</height>
       </rect>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <column>
       <property name="text">
        <string>ID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>F_ Carga</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Dias</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Vence</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>ID _U</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Obs</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Nro T</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Cerrado</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Detalle</string>
       </property>
      </column>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>891</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>22</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Recordatorios</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnEditar">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>470</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);
border-radius: 15px;</string>
    </property>
    <property name="text">
     <string>Editar</string>
    </property>
   </widget>
   <zorder>btnEditar</zorder>
   <zorder>tablaVenci</zorder>
   <zorder>btnAnadir</zorder>
   <zorder>frame</zorder>
   <zorder>label_9</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
